projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -990,7 +1035,7 @@ public function render_field( $type, $id, $label, $value, $class, $placeholder,
 				break;
 		}
 
-		if ( ! empty( $form_style ) && $form_style !== 'default' && ! in_array( $type, array( 'checkbox', 'consent' ), true ) ) {
+		if ( ! empty( $form_style ) && $form_style !== 'default' && ! in_array( $type, array( 'checkbox', 'checkbox-multiple', 'radio', 'consent' ), true ) ) {
```

Comment:
I'm not sure why the label is duplicated at the end of the field in some cases. The order of elements should usually be handled in CSS.

I've removed both the single choice and multiple choice inputs from the check: the outline style is fully handled in CSS; the animated style doesn't seem to be implemented anyway; I can't see the Below style in the UI but I've never seen checkboxes or radio buttons with a label below them.